include/constants/trainers.h

```diff
@@ -117,95 +117,95 @@
+#define FACILITY_CLASS_BEAUTY               0x14
+#define FACILITY_CLASS_RICH_BOY             0x15
+#define FACILITY_CLASS_EXPERT_F             0x16
+#define FACILITY_CLASS_POKEMANIAC           0x17
+#define FACILITY_CLASS_MAGMA_GRUNT_F        0x18
+#define FACILITY_CLASS_GUITARIST            0x19
+#define FACILITY_CLASS_KINDLER              0x1a
+#define FACILITY_CLASS_CAMPER               0x1b
+#define FACILITY_CLASS_PICNICKER            0x1c
+#define FACILITY_CLASS_BUG_MANIAC           0x1d
+#define FACILITY_CLASS_PSYCHIC_M            0x1e
+#define FACILITY_CLASS_PSYCHIC_F            0x1f
+#define FACILITY_CLASS_GENTLEMAN            0x20
+#define FACILITY_CLASS_ELITE_FOUR_SIDNEY    0x21
+#define FACILITY_CLASS_ELITE_FOUR_PHOEBE    0x22
+#define FACILITY_CLASS_LEADER_ROXANNE       0x23
+#define FACILITY_CLASS_LEADER_BRAWLY        0x24
+#define FACILITY_CLASS_LEADER_TATE_AND_LIZA 0x25
+#define FACILITY_CLASS_SCHOOL_KID_M         0x26
+#define FACILITY_CLASS_SCHOOL_KID_F         0x27
+#define FACILITY_CLASS_SR_AND_JR            0x28
+#define FACILITY_CLASS_POKEFAN_M            0x29
+#define FACILITY_CLASS_POKEFAN_F            0x2a
+#define FACILITY_CLASS_YOUNGSTER            0x2b
+#define FACILITY_CLASS_CHAMPION_WALLACE     0x2c
+#define FACILITY_CLASS_FISHERMAN            0x2d
+#define FACILITY_CLASS_CYCLING_TRIATHLETE_M 0x2e
+#define FACILITY_CLASS_CYCLING_TRIATHLETE_F 0x2f
+#define FACILITY_CLASS_RUNNING_TRIATHLETE_M 0x30
+#define FACILITY_CLASS_RUNNING_TRIATHLETE_F 0x31
+#define FACILITY_CLASS_SWIMMING_TRIATHLETE_M 0x32
+#define FACILITY_CLASS_SWIMMING_TRIATHLETE_F 0x33
+#define FACILITY_CLASS_DRAGON_TAMER         0x34
+#define FACILITY_CLASS_NINJA_BOY            0x35
+#define FACILITY_CLASS_BATTLE_GIRL          0x36
+#define FACILITY_CLASS_PARASOL_LADY         0x37
+#define FACILITY_CLASS_SWIMMER_F            0x38
+#define FACILITY_CLASS_TWINS                0x39
+#define FACILITY_CLASS_SAILOR               0x3a
+#define FACILITY_CLASS_PKMN_TRAINER_WALLY   0x3b
+#define FACILITY_CLASS_PKMN_TRAINER_BRENDAN 0x3c
+#define FACILITY_CLASS_PKMN_TRAINER_BRENDAN_2  0x3d
+#define FACILITY_CLASS_PKMN_TRAINER_BRENDAN_3  0x3e
+#define FACILITY_CLASS_PKMN_TRAINER_MAY     0x3f
+#define FACILITY_CLASS_PKMN_TRAINER_MAY_2   0x40
+#define FACILITY_CLASS_PKMN_TRAINER_MAY_3   0x41
+#define FACILITY_CLASS_PKMN_BREEDER_M       0x42
+#define FACILITY_CLASS_BUG_CATCHER          0x43
+#define FACILITY_CLASS_PKMN_RANGER_M        0x44
+#define FACILITY_CLASS_PKMN_RANGER_F        0x45
+#define FACILITY_CLASS_MAGMA_LEADER_MAXIE   0x46
+#define FACILITY_CLASS_LASS                 0x47
+#define FACILITY_CLASS_YOUNG_COUPLE         0x48
+#define FACILITY_CLASS_OLD_COUPLE           0x49
+#define FACILITY_CLASS_SIS_AND_BRO          0x4a
+#define FACILITY_CLASS_STEVEN               0x4b
+#define FACILITY_CLASS_SALON_MAIDEN_ANABEL  0x4c
+#define FACILITY_CLASS_DOME_ACE_TUCKER      0x4d
+#define FACILITY_CLASS_RED                  0x4e
```

Comment:
Why did you remove the PKMN_TRAINER part in these four but leave it in the WALLY/BRENDAN/MAY ones above?

Reply:
Oversight, fixed.